### PR TITLE
Fix axa-text bug

### LINKED ISSUE #1790: Axa-text can delete its children
The axa-text component implementation violates a [documented restriction on `render()` methods in lit-element](https://lit-element.polymer-project.org/guide/lifecycle#render) by not returning anything.

As a result, sometimes the children of this component are deleted _after_ an initially successful render.

Fix this wrong behaviour in an lit-element compliant fashion.

### PR DESCRIPTION
**Pull request by markus-walther**
Fixes #1790.

### CODE REVIEW

File: src/components/10-atoms/text/react/demo.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Readme from '../README.md';
+import Changelog from '../CHANGELOG.md';
+
+storiesOf('Components|Atoms/Text/React/Demos', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+    changelog: Changelog,
+  })
```
Comment (danyball) [inline src/components/10-atoms/text/react/demo.jsx:16]: You can disable knobs tab here.

File: src/components/10-atoms/text/ui.test.js [hunk 2]
```diff
@@ -184,3 +184,62 @@ test('should update pure text dynamically and wrap in <p>', async t => {
...
+
+fixture('Text - React')
+  .page(
+    `${host}/iframe.html?id=components-atoms-text-react-demos--feature-text-with-dynamic-children-under-react`
+  )
+  .beforeEach(async t => {
+    await t.resizeWindow(800, 600);
+  });
+
+test('should update dynamically for pure and HTML texts', async t => {
```
Comment (danyball) [inline src/components/10-atoms/text/ui.test.js:196]: Dont you want to split this huge test into 2? One for pure and the other for html texts.

**Review by danyball — DISMISSED** [review]

File: src/components/10-atoms/text/ui.test.js (hunk 2 above)
Comment (markus-walther) [reply src/components/10-atoms/text/ui.test.js:196]: I interpret this request as "don't you want to make this test shorter, more readable" and DRYify it internally. Hope that satisfies the wish you had here.

Done.

File: src/components/10-atoms/text/react/demo.jsx (hunk 1 above)
Comment (markus-walther) [reply src/components/10-atoms/text/react/demo.jsx:16]: ... via `knobs:{disabled: true}` - thanks for the tip! 

Done.

**Review by danyball — APPROVED** [review]

**Pull request merged**